tools/tf2openapi/cmd/main.go

```diff
@@ -10,9 +10,12 @@ import (
 	"os"
 )
 
+const defaultSigDefKey = "serving_default"
```

Comment:
In general main.go is not a good place to put those constants. One const may be fine.

Reply:
Moving const to `generate.go`

Reply:
I use this as the default SignatureDef key if the user doesn't supply any values. 

The alternative would be moving the const to `generate.go` and using this value if the param `sigDefKey` of `GenerateOpenAPI` is "", so I can do that instead.